command/helper/config.go

```diff
@@ -94,7 +90,15 @@ func (c *Config) BuildConfig() (*server.Config, error) {
 	conf.Chain = cc
 	conf.Seal = c.Seal
 	conf.DataDir = c.DataDir
+	// Set the secrets manager config if it was passed in
+	if c.Secrets != "" {
+		secretsConfig, readErr := secrets.ReadConfig(c.Secrets)
+		if readErr != nil {
+			return nil, fmt.Errorf("unable to read config file, %v", readErr)
+		}
 
+		conf.SecretsManager = secretsConfig
+	}
```

Comment:
You read the config here, but it was never saved to this configuration struct.

Please notice you're only saving it into a temporary var here, but that's about it:
https://github.com/0xPolygon/polygon-sdk/blob/f4d9896f0b2dc70bb3226f0149c6fbb0676dcd3a/command/helper/helper.go#L432

Reply:
Thanks for the catch , fixed it [here](https://github.com/0xPolygon/polygon-sdk/pull/270/commits/0c7ba6270159d89fcaa6a3f9bf6e3e8548ed6481)